<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefab</class>
 <widget class="QDialog" name="prefab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background:#353637}
QWidget{background:#4d4e4e}
QPushButton{background:#f6bd40;color:white;font:18px}
QPushButton:hover{background:#f9d483;}
QLabel#lbl_price{color:#f6bd40}
QLabel#lbl_start,#lbl_end{color:rgb(85, 255, 255);font:14px}
QLabel#lbl_time_start,#lbl_time_end{color:white;font:20px}
QLabel#lbl_before,#lbl_flghtid{color:#3fc6ff}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="lbl_flightid">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>39</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_time_start">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>06:45</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_time_end">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>09:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_start">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>白云</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_end">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>虹桥</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_during">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2小时15分</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_arrow">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../res.qrc">:/res/arrow_right.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buy">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>订票</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>67</width>
        <height>36</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_price1">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>¥</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_price">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>450</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lbl_before">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>空客A01</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
